curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-color-markers-set/step-082.md

```diff
@@ -0,0 +1,124 @@
+---
+id: 61addaf7e83988b59a7d18ff
+title: Step 82
+challengeType: 0
+dashedName: step-82
+---
+
+# --description--
+
+The `border-left` shorthand property lets you to set the left border's width, style, and color at the same time.
+
+Here is the syntax:
+
+```css
+border-left: width style color;
+```
+
+In the `.sleeve` CSS rule, replace the `border-left-width`, `border-left-style`, and `border-left-color` properties with the `border-left` shorthand property. The values for the width, style, and color of the left border should be the same.
+
+# --hints--
+
+Your `.sleeve` CSS rule should not have a `border-left-width` property and value.
+
+```js
+assert.isEmpty(new __helpers.CSSHelp(document).getStyle('.sleeve')?.borderLeftWidth);
+);
+```
+
+Your `.sleeve` CSS rule should not have a `border-left-style` property and value.
+
+```js
+assert.isEmpty(new __helpers.CSSHelp(document).getStyle('.sleeve')?.borderLeftStyle);
+```
+
+Your `.sleeve` CSS rule should not have a `border-left-color` property and value.
+
+```js
+assert.isEmpty(new __helpers.CSSHelp(document).getStyle('.sleeve')?.borderLeftColor);
```

Comment:
````suggestion
assert(!/border-left-width/g.test(code));
```

Your `.sleeve` CSS rule should not have a `border-left-style` property and value.

```js
assert(!/border-left-style/g.test(code));
```

Your `.sleeve` CSS rule should not have a `border-left-color` property and value.

```js
assert(!/border-left-color/g.test(code));
````